### ISSUE TITLE
Accessibility: Missing form labels on Submit Poem form

### ISSUE THREAD

**Issue by mattfromseattle** [suwebdev/great-sunrise#5]
**Issue:** Users with vision impairment should be able to use screen readers to review and fill out forms on the site. Currently, the Submit Poem form is missing the necessary labels for a screen reader to recite what field the visitor is on.

**User Story:** https://github.com/suwebdev/great-sunrise/wiki/Accessibility-User-Stories 

**Priority:** HIGH
This is a HIGH priority issue as users with visual impairment are unable to navigate the form necessary to submit new poems to the site. In order to be compliant with accessibility, we must have this fixed.

**Considerations:**
- All necessary fields will have matching "for" and "id" tags
- Screen readers such as JAWS and VoiceOver for Mac should be supported
- Images of text should be avoided

**User Story:** As a screen-reader user, I want to be able to know what each form label is for each form field so I can enter the correct information in the correct field within the form.

**Acceptance Criteria:**
- Labels shall be provided to identify all form controls, including text fields, checkboxes, radio buttons, and drop-down menus.
- label elements shall be used to associate text with the corresponding form elements. The for attribute of the label must accurately match the id of the form control.

**Estimated Sized:** **sm** (1 day)

**Comment by mattfromseattle** [comment]
Pull Request #22 created to address this issue.
